src/main/java/tc/oc/pgm/rotation/FixedPGMMapOrder.java

```diff
@@ -105,25 +109,15 @@ private int getMapPosition(PGMMap map) {
   }
 
   private void rotate() {
-    if (position + 1 >= maps.size()) {
-      position = 0;
-    } else {
-      movePosition(1);
-    }
+    movePosition(1);
   }
 
   public void movePosition(int positions) {
-    if (position + positions >= maps.size()) {
-      position = (position + positions) % maps.size();
-    } else {
-      position = position + positions;
-    }
+    position = (position + positions) % maps.size();
   }
 
   PGMMap getMapInPosition(int position) {
-    if (position == maps.size()) {
-      return getMapInPosition(0);
-    } else if (position > maps.size()) {
+    if (position > maps.size()) {
```

Comment:
```suggestion
    if (position >= maps.size()) {
```
To properly restrict this, it should be invalid if == to size too.

Reply:
Fixed! https://github.com/Electroid/PGM/pull/120/commits/ee1be6fa4580db48098150a6d586a056038b5afd